RequestKit.xcodeproj/project.pbxproj

```diff
@@ -867,6 +868,7 @@
 				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks @loader_path/Frameworks";
 				SKIP_INSTALL = YES;
 				SWIFT_OPTIMIZATION_LEVEL = "-Owholemodule";
+				SWIFT_VERSION = 3.0;
```

Comment:
thanks for the feedback! I just updated the project settings once more, not quite sure if that's the desired outcome though... let me know if I should update it again :)

    SWIFT_VERSION = "";


<img width="491" alt="screen shot 2016-11-15 at 21 51 48" src="https://cloud.githubusercontent.com/assets/4058327/20336349/40ee6e8c-ab7e-11e6-8803-a6a7df501c50.png">